components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -647,22 +666,34 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
         }
     }
 
+    private int handleLimitEqualsNULL(Integer limit){
+
+        // Limit equal to null implies return all users. Return all users scenario handled by the following methods by
```

Comment:
line exceeds max length

Reply:
The formatter does not recognize it has an exceeded. It seems here that it exceeds the limit. But in IDEA it isn't.